### DISCUSSION TITLE
Leveraging `opsuite` package to visualize body maps of PBPK models

### DISCUSSION THREAD

**Discussion by pchelle** [Open-Systems-Pharmacology/Forum#1514] [Feature Requests & Ideas]
Sometimes it is nice to visualize and perform comparisons of PBPK models or model features using body maps (named **anatograms**).
For instance, comparing between populations (males vs females, adults vs infants, human vs murine, etc.) or between administrations (oral, dermal, IV, IP, etc.)

Since the PBPK models can be exported to R with `{ospsuite}` package, it can be integrated to the [`{gganatogram}`](https://r-charts.com/miscellaneous/gganatogram/) package to create such body maps.
Then, the body map can be integrated to a website, shiny app or even video to make nice visuals of the models.
The main barrier being that your PK-Sim paths may need to be mapped to the anatogram body parts using a dictionary.

[Here](https://github.com/Open-Systems-Pharmacology/Forum/files/11344195/test-ospsuite-anatogram.zip) is a very simple example using a dummy model along with a screenshot below:

![image](https://user-images.githubusercontent.com/45343665/234882098-123b814f-35a5-41cc-a3f3-0beefa07ff74.png)

**Comment by PavelBal** [comment]
Nice!
